### PR TITLE
Updating Grafana deployment specific to Influxdb v0.8.9.

### PR DESCRIPTION
**Pull request by vishh**

### CODE REVIEW

**Comment by vishh** [comment]
cc @mvdan

**Comment by mvdan** [comment]
Just noticed that we have a file with a space, `docs/Grafana Dashboard.json`. That should be `docs/grafana-dashboard.json`.

File: docs/Grafana Dashboard.json [hunk 1]
```diff
@@ -145,8 +116,9 @@
                 }
               ],
               "groupByTags": [],
+              "hide": false,
```
Comment (mvdan) [inline docs/Grafana Dashboard.json]: I can't help but notice that some changes are simply moving lines around, like `hide` and `interval`. Are these changes automated or manual?

Comment (vishh) [inline docs/Grafana Dashboard.json]: automated.

Comment (mvdan) [inline docs/Grafana Dashboard.json]: Fair enough then.

**Comment by mvdan** [comment]
Only that filename nit, otherwise LGTM.

**Comment by vishh** [comment]
Done!

**Comment by vishh** [comment]
Updated the filename

**Pull request merged**